cdap-standalone/src/main/resources/cdap-site.xml

```diff
@@ -56,6 +58,14 @@
     </description>
   </property>
 
+  <property>
+    <name>preview.cache.size</name>
+    <value>10</value>
+    <description>
+      The size of the preview cache.
```

Comment:
nit: `Number of preview runs for which data will be stored in memory. When size exceeds, data for the oldest run will be removed from memory and no longer retrievable.`